xml/containers-build.xml

```diff
@@ -0,0 +1,171 @@
+    daemon to do it. Another important feature of Podman are
+    <emphasis role="strong">rootless container builds</emphasis>. By using user
+    namespaces, which provide an isolation mechanism for Linux processes,
+    Podman avoids requiring root privileges to create and run containers.
+   </para>
+   <para>
+    For instructions on how to install and use Podman see
+    <link xlink:href="https://podman.io/">the Podman Documentation</link>.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="id-buildah">
+   <title>Buildah</title>
+   <para>
+    Buildah facilitates building <emphasis role="strong">Open Containers
+    Initiative</emphasis> (OCI) container images. It is a complimentary tool to
+    Podman (see<xref linkend="podman"/> ), in fact <literal>podman
+    build</literal> uses Buildah to perform container image builds. Images can
+    be built from already existing images, from scratch, and using Dockerfiles.
+    OCI images built using the <emphasis role="strong">Buildah command line
+    tool</emphasis> (CLI) and the underlying OCI based technologies (e.g.
+    containers/image and containers/storage) are portable and can therefore run
+    in a Docker environment.
+   </para>
+   <para>
+    For instructions on how to install and use Buildah see
+    <link xlink:href="https://buildah.io/">the Buildah Documentation</link> and
+    <link xlink:href="https://github.com/containers/buildah/blob/master/docs/tutorials">tutorials</link>.
+   </para>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="id-suse-internal-build-tools">
+  <title>SUSE-Internal Build Tools</title>
+
+  <sect2 xml:id="id-open-build-service">
+   <title>Open Build Service</title>
+   <para>
+    The <emphasis role="strong">Open Build Service</emphasis> (OBS) provides
+    free infrastructure for building and storing RPM packages including various
+    container formats. An important component of the OBS is also the
+    <emphasis role="strong">OBS Container Registry</emphasis>. The
+    <link xlink:href="https://registry.opensuse.org/cgi-bin/cooverview">OBS
+    Container Registry</link> provides a detailed listing of all container
+    images built by the OBS complete with command line code to pull the images
+    into your local docker environment. The OBS &opensuse;
+    <link xlink:href="https://build.opensuse.org/image_templates">container
+    image templates</link> can be modified to specific needs and are the
+    easiest way to create your own container branch. Container images can
+    either be built with <emphasis role="strong">native Docker tools</emphasis>
+    - based on an existing image using a Dockerfile. Or they can be built from
+    scratch using the <emphasis role="strong">&kiwi; imaging
+    solution</emphasis>.
+   </para>
+   <para>
+    Instructions on how to build images on OBS can be found
+    <link xlink:href="https://openbuildservice.org/2018/05/09/container-building-and-distribution/">here</link>.
```

Comment:
"Here" is pretty bad as link text -- maybe just use `at <link xlink:href="https://openbuildservice.org/2018/05/09/container-building-and-distribution/"/>`